.ci/build-in-ci.ps1

```diff
@@ -1,5 +1,5 @@
-choco install docfx -y --version 2.48.1
-docfx --warningsAsErrors docfx.json
+nuget install docfx.console
+.\docfx.console*\tools\docfx.exe --warningsAsErrors docfx.json
```

Comment:
```suggestion
.\docfx.console*\tools\docfx.exe docfx.json
```

Although not the ideal solution this would fix the build process and allow recent documentation updates to be published.